src/backend/api/Fusion.Resources.Domain/Models/QueryRelevantOrgUnit.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace Fusion.Resources.Domain.Models
+{
+    public class QueryRelevantOrgUnit
+    {
+        public string? FullDepartment { get; set; }
+        public List<string> Reasons { get; set; } = new();
+        public string? Name { get; set; }
+        public string? SapId { get; set; }
```

Comment:
Not a problem, but I would suggest to place the identifier column on top (SapId)